app/routes/users+/$username_+/articles.$articleId.tsx

```diff
@@ -0,0 +1,221 @@
+import { getFormProps, useForm } from '@conform-to/react'
+import { parseWithZod } from '@conform-to/zod'
+import { invariantResponse } from '@epic-web/invariant'
+import {
+	json,
+	type LoaderFunctionArgs,
+	type ActionFunctionArgs,
+} from '@remix-run/node'
+import {
+	Form,
+	Link,
+	useActionData,
+	useLoaderData,
+	type MetaFunction,
+} from '@remix-run/react'
+import { formatDistanceToNow } from 'date-fns'
+import { z } from 'zod'
+import { GeneralErrorBoundary } from '~/components/ErrorBoundary.js'
+import { floatingToolbarClassName } from '~/components/floating-toolbar.tsx'
+import { ErrorList } from '~/components/forms.tsx'
+import { Button } from '~/components/ui/button.tsx'
+import { Icon } from '~/components/ui/icon.tsx'
+import { StatusButton } from '~/components/ui/status-button.tsx'
+import { requireUserId } from '~/utils/auth.server.ts'
+import { prisma } from '~/utils/db.server.ts'
+import { getArticleImgSrc, useIsPending } from '~/utils/misc.tsx'
```

Comment:
You've forgotten to create and export a `getArticleImgSrc` function from `'~/utils/misc.tsx'`. Remind yourself how to do this via the [Code Monster section](https://gedsargent.github.io/code-monster/guides/epic-news/prisma/modify-project-structure/#add-getarticleimgsrc-function) we looked at together.

You then really need to pick up the pace of working as you are a long way behind, Raf.